server/migrations/20190204141619-create-users.js

```diff
@@ -0,0 +1,38 @@
+
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('users', {
+    userId: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.UUID
+    },
```

Comment:
@jsamchineme thanks for the feedback however I would suggest that we as a team discuss on what to use as could we write a function to generate our own UUID, use UUIDV1 or UUIDV4.
http://docs.sequelizejs.com/variable/index.html

Reply:
Also 'Only` articleId` and `userId` should be` UUID`. All other tables primary key should be `INTEGER`, autoIncrementing.' I decided to use UUID for other models to maintain consistency since the aim is to maintain uniqueness. Also this prevents the error that may occur when making relationships  between UUID and integer as UUID has more precision than Integer.